src/components/NodeEditable.js

```diff
@@ -63,7 +63,7 @@ class NodeEditable extends Component {
         },
         e => {
           const errorText = SHARED.parser.getExceptionMessage(e);
-          say(errorText);
```

Comment:
`say` logs to the console anyway

Reply:
But it also reads out loud the entirety of the error message! That's an awful user experience.